ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -949,26 +963,34 @@ static GstCaps *
 _ts_get_gstcaps_from_conf (GstTensorSrcTIZENSENSOR * self)
 {
   const GstTensorInfo *spec;
-  gchar *tensor;
   GstCaps *retval;
+  GstTensorConfig tensor_config;
```

Comment:
Do you really need to use this extra config?

Reply:
To make use of `gst_tensor(s)_caps_from_config` the config variable is needed. Else, caps may defined from hard-coded string. So I introduce the `tensor_config`.

And, I moved the declaration of `tensor_config` into else {} statement. Please check!

Reply:
Yes, but I think the codes seem to be redundant (e.g., setting rate). Is it possible to reuse `tensors_info` because `num_tensors = 1`?

Reply:
`GstTensorsConfig` consists of `GstTensorsInfo` and rate info, not of `GstTensorConfig`.  Thus, IMHO, the two config variables (`tensor_config` and `tensors_config`) are necessary. If you have a neat way to address, please let me know..

Reply:
Ahh got it